react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const pokeList = pokedex.map(pokedex =>
```

Comment:
Although it works fine I would recommend using a different name for your callback function parameter to avoid confusion, something like `pokemon` or something similar. We usually use the singular of whatever is being looped through.